examples/nodejs-simple/Dockerfile

```diff
@@ -35,6 +35,9 @@ RUN apt-get update && apt-get install -y nodejs
 
 WORKDIR /home/server
 
+COPY ./sdks/nodejs sdks/nodejs
+RUN cd sdks/nodejs && \
```

Comment:
I'm not sure why we have `npm rebuild` below so this may be a good time to remove it and on this line too, but it won't do anything bad.

`npm rebuild` calls `npm build` which is an internal command to npm. The same command is called from `npm install` already as far as I could gather, so is redundant. Information is a bit scarce on this.

Another potential error in this file is that we are using `npm install` instead of `npm ci`. The former can override `package-lock.json` and also takes slightly longer to run. Generally `npm install` should only be run by users and `npm ci` by automation. In this case I don't think we are running tests as part of the Docker build so do not need the dev dependencies and so can use `npm ci --production`

So these changes are fine but we are duplicating lines that could be better

Reply:
Thank you for your great review!!

I updated and tested locally. This works fine.